src/screens/HyperSwitch/UserManagement/ShowUserData.res

```diff
@@ -6,23 +6,9 @@ external typeConversion: array<Js.Nullable.t<UserRoleEntity.userTableTypes>> =>
 
 module UserHeading = {
   @react.component
-  let make = (~infoValue: UserRoleEntity.userTableTypes, ~userId) => {
```

Comment:
Hey @shubham0804 can you revert this file also as this in WIP. So let it be the warnings in this.

Reply:
Hi @PritishBudhiraja. Done